themes/digital.gov/layouts/partials/core/featured-resource.html

```diff
@@ -1,66 +1,104 @@
+{{ $externalLink := "" }}
+{{ $isExternal := false }}
 
-<!-- prettier-ignore -->
-Community resource paths:
-  - communicators
-  - multilingual
-  - plain-language
-  - social-media
-  - user-experience
-  - web-analytics-and-optimization
-  - web-managers-forum
-*/}}
+{{/* Check if page is an external link, otherwise return a hugo page object */}}
+{{ if or (strings.HasPrefix $resource.link "https://") (strings.HasPrefix $resource.link "http://") }}
+  {{ $isExternal = true }}
+  {{ $title = $resource.title }}
+  {{ $summary = $resource.summary }}
+  {{ $kicker = $resource.kicker }}
+  {{ $externalLink = $resource.link }}
+{{ else }}
+  {{ $page := .Site.GetPage $resource.link }}
+  {{ $title = $page.Title }}
+  {{ $internalLink = $page.Permalink }}
+  {{ $summary = $page.Params.summary }}
+  {{ $image = $page.Params.image }}
 
-{{- $resourcePath := .resourcePath -}}
-{{- $kicker := .kicker -}}
-{{- if $resourcePath -}}
-  {{- $resource := .Site.GetPage $resourcePath -}}
-  {{- if $resource -}}
+  {{/* Check if kicker is present as a custom key, otherwise use page parameter to set */}}
+  {{ if $resource.kicker }}
+    {{ $kicker = $resource.kicker }}
+  {{ else }}
+    {{ if $page.Params.kicker }}
+      {{ $kicker = $page.Params.kicker }}
+    {{ end }}
+  {{ end }}
+{{ end }}
 
-    <div class="dg-featured-resource">
-      <a
-        href="{{- $resource.Permalink | relURL -}}"
-        class="dg-featured-resource__body"
-      >
-        <div class="dg-featured-resource__text">
-          <p class="dg-featured-resource__text-kicker">
-            {{- if $kicker -}}
-              {{- $kicker | markdownify -}}
-            {{- else -}}
-              Featured Resource
-            {{- end -}}
-          </p>
-          <h2 class="dg-featured-resource__text-title">
-            {{- $resource.Params.title -}}
-          </h2>
-          <p class="dg-featured-resource__text-description">
-            {{- $resource.Params.summary -}}
-          </p>
+{{- if $resource -}}
+  <div class="dg-featured-resource">
+    <a
+      href="{{ if $isExternal }}
+        {{ $externalLink }}
+      {{ else }}
+        {{- $internalLink | relURL -}}
+      {{ end }}"
+      class="dg-featured-resource__body"
+    >
+      <div class="dg-featured-resource__text">
+        <p class="dg-featured-resource__text-kicker">
+          {{- if $kicker -}}
+            {{- $kicker | markdownify -}}
+          {{- else -}}
+            Featured Resource
+          {{- end -}}
+        </p>
+        <h2 class="dg-featured-resource__text-title">
+          {{ if $title }}
+            {{- $title -}}
+          {{ end }}
+        </h2>
+        <p class="dg-featured-resource__text-description">
+          {{ if $summary }}
+            {{- $summary -}}
+          {{ end }}
+        </p>
+      </div>
+      {{ if $image }}
+        <div class="dg-featured-resource__image">
+          {{- partial "core/img-flexible.html" (dict "Site" .Site "src" $image "small" "true") -}}
```

Comment:
We're always passing `.Site` to this component. 

Anyway to make that the default instead of passing the same param every time?

Reply:
Good call out, but we are stuck with having to pass in `.Site` to partials/components with the current code.

We could refactor this partial to re-reference the global scope:

```go

{{ $site := . }}

```

This would require additional refactoring if we should pursue this.